dotCMS/src/main/java/com/dotcms/publishing/BundlerUtil.java

```diff
@@ -116,6 +115,7 @@ public static void objectToXML ( Object obj, File f, boolean removeFirst ) {
             f.delete();
 
         XStream xstream = new XStream( new DomDriver("UTF-8") );
+        OutputStreamWriter writer = null;
 
         try {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change this "try" to a try-with-resources. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2093)

Reply:
not valid